VersionInfo.cpp

```diff
@@ -181,15 +185,176 @@ static void fill_compiler_info(unichar* s) {
 
 
 #ifdef __DATE__
+// see https://stackoverflow.com/questions/11697820/how-to-use-date-and-time-predefined-macros-in-as-two-integers-then-stri
```

Comment:
Use instead the macro: `UNITEX_VERSION_BUILD_DATE` already defined on [Version.h](UnitexGramLab/unitex-core/blob/master/Version.h). FYK, the value is assigned at building time as `date '+%B %d, %Y'` by [Vinber](UnitexGramLab/vinber-backend/blob/master/vinber.sh#L46). Other related macros are:

`UNITEX_VERSION_BUILD_DAY `
`UNITEX_VERSION_BUILD_MONTH `
`UNITEX_VERSION_BUILD_YEAR`
`UNITEX_VERSION_BUILD_TIMESTAMP `